.github/workflows/commit-validation.yml

```diff
@@ -0,0 +1,42 @@
+name: "Commit message validation"
```

Comment:
Could we add some top-level comment description of what this pipeline does (and probably point to our conventional commits guidelines doc on the wiki)?

Reply:
Done.